### ISSUE TITLE
QMOpenGraphCache NSAssert

### ISSUE THREAD

**Issue by sin2** [QuickBlox/q-municate-services-ios#336]
Updated to the newest version, and I am hitting this assert.
```
+ (QMOpenGraphCache *)instance {
    
    NSAssert(_openGraphCache, @"You must first perform @selector(setupDBWithStoreNamed:)");
    return _openGraphCache;
}
```

I am not using or have setup up the core data services.
I cannot revert to 0.4.6, or 0.4.5

**Comment by ghost** [comment]
Hi @sin2, Use `setupDBWithStoreNamed` before use instance

```objc
        //Open Graph Service init
        [QMOpenGraphCache setupDBWithStoreNamed:kQMOpenGraphCacheNameKey
                     applicationGroupIdentifier:@"your-app-group-identifier-or-nil"];
```
See example:
https://github.com/QuickBlox/q-municate-ios/blob/master/Q-municate/Classes/Core/QMCore/QMCore.m#L64

**Comment by sin2** [comment]
I am not using this instance. It is being used by the default implementation of qmservicemanager in the latest version. If I try to run setupdbwithstorenamed I get another assert saying it must be overloaded.

**Comment by sin2** [comment]
Also this assert gets triggered when I run a disconnect when a user logs out.

**Comment by ghost** [comment]
fixed 0.5.1
